packages/origin-ui-core/src/components/Account/UserProfile/UserDataEditForm.tsx

```diff
@@ -0,0 +1,260 @@
+                                                    success
+                                                        ? 'user.feedback.confirmationEmailResent'
+                                                        : 'user.feedback.confirmationEmailResentFailed'
+                                                );
+
+                                                showNotification(
+                                                    message,
+                                                    success
+                                                        ? NotificationType.Success
+                                                        : NotificationType.Error
+                                                );
+                                            }}
+                                        >
+                                            {t('user.actions.resendConfirmationEmail')}
+                                        </Button>
+                                    )}
+
+                                    <Field
+                                        label={t('user.properties.kycStatus')}
+                                        type="kycStatus"
+                                        name="kycStatus"
+                                        component={TextField}
+                                        variant="filled"
+                                        fullWidth
+                                        disabled={true}
+                                        value={values.kycStatus}
+                                        className="mt-3"
+                                    />
+                                </Grid>
+                            </Grid>
+                            {isEditing && (
+                                <Button
+                                    style={{ marginRight: 10 }}
+                                    type="button"
+                                    variant="contained"
+                                    color="primary"
+                                    className="mt-3 right"
+                                    disabled={
+                                        !(
+                                            touched.firstName ||
+                                            touched.lastName ||
+                                            touched.telephone
+                                        )
+                                    }
+                                    onClick={async () => {
+                                        await formikProps.validateForm();
+                                        await formikProps.submitForm();
+                                        setIsEditing(false);
+                                    }}
+                                >
+                                    {t('user.actions.save')}
+                                </Button>
+                            )}
+                            {isEditing && (
```

Comment:
Same here. If you change some data and then click Cancel - your Save and Cancel buttons will no longer be disabled by default.

Reply:
fixed